src/formaters/formatToCurrency.js

```diff
@@ -0,0 +1,64 @@
+/**
+ *
+ * Formats number (integer or float) to currency
+ * @param {number} number
+ *
+ **/
+
+export default function formatToCurrency(number, numDecimals) {
+  if (typeof number !== 'number' || isNaN(number)) return false
```

Comment:
@JoaoFGuiomar overkill maybe? `isNaN` is unnecessary :P 

Also since this is a formater it should throw an `TypeError`